lessons/lesson-3/project/how-to-add-audio-feedback.md

```diff
@@ -1,2 +1,45 @@
-# How to add audio feedback?
```

Comment:
@aprilspeight please keep the format and don't capitalize. The reason is, some titles are long like: "What do I need to do to make sure Android, iOS and HoloLens are using the same point as my anchor?". It doesn't look good when all capital. Thanks!